### ISSUE TITLE
[Bug]: i got en error when i run function: failed on pre-loading / post-running of the function

### ISSUE THREAD

**Issue by kuanghualuoye** [mlrun/mlrun#4494]
### MLRun Version checks

- [X] I have checked that this issue has not already been reported.

- [X] I have confirmed this bug exists on the latest version of MLRun CE.

### Reproducible Example

```python
import mlrun

project = mlrun.get_or_create_project('aaaa', save=True)
project.set_function('src/ingest_data.py',
                     name='ingest',
                     handler='load_data',
                     kind='job',
                     image='aaa/mlrun-base')
ingest = project.run_function('ingest', inputs=params, verbose=True)
```

### Issue Description

i run the code above got en error

![image](https://github.com/mlrun/mlrun/assets/43460848/ce7e297d-c707-4d03-840e-24d01a074f1c)
<img width="1302" alt="image" src="https://github.com/mlrun/mlrun/assets/43460848/d816606e-a2fd-40c2-a4eb-8193974a917f">

but i run it in jupyter lab successfully....

### Expected Behavior

please help me fix it or how to find the reason

### Installation OS

Linux

### Installation Method

Kubernetes

### Python Version

3.9.15

### MLRun Version

1.2.1

**Comment by alonmr** [comment]
Hey @kuanghualuoye, thank you for reporting.
MLRun CE 0.6.2 is shipped with MLRun 1.4.0, you specified that the bug exist on latest MLRun CE but the MLRun version you specified is 1.2.1. Can you please clarify?
1 more thing to note is that MLRun 1.2.1 does not support python 3.9. It was introduced in MLRun 1.3.0.

**Comment by alonmr** [comment]
@kuanghualuoye please reopen id still an issue with latest.
